server/service/src/sync/sync_serde.rs

```diff
@@ -75,6 +76,15 @@ pub fn naive_time<'de, D: Deserializer<'de>>(d: D) -> Result<NaiveTime, D::Error
         .unwrap_or(NaiveTime::from_hms_opt(0, 0, 0).unwrap()))
 }
 
+pub fn empty_str_or_i32<'de, D: Deserializer<'de>>(d: D) -> Result<i32, D::Error> {
+    let value = Value::deserialize(d)?;
```

Comment:
The max_lines sent from OG is either: 
- A string if empty
OR
- A number. 

Easier to just parse it here..